Quadriga/src/main/java/edu/asu/spring/quadriga/validator/AddProjectValidator.java

```diff
@@ -31,7 +31,6 @@ public boolean supports(Class<?> arg0) {
      * 
      * @param obj
      * @param err
-     * @author Karthikeyan Mohan

```

Comment:
I had to remove it on PublicpageValidator.java file. But, accidentally I did on this java file as well, I will add it back.